python-api/src/api/llm/router.py

```
@@ -0,0 +1,447 @@
+            )
+
+    except WebSocketDisconnect:
+        logger.info("WebSocket /llm/ws/chat disconnected")
+    except Exception as exc:
+        logger.error("WebSocket error: %s", exc, exc_info=True)
+        try:
+            await websocket.send_text(
+                WSOutgoingMessage(event="error", error="Internal server error.").model_dump_json()
+            )
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)